force-app/main/default/classes/SOQL.cls

```diff
@@ -50,33 +61,58 @@ public with sharing class SOQL {
      * @return A list of Contact Ids.
      * Update the corresponding test class and testGetContactIdsByLastName method to cover this method
      */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+    public static List<Id> getContactIdsByLastName() {
+        List<Contact> contacts = [
+            SELECT Id 
+            FROM Contact 
+            WHERE LastName 
+            IN ('Smith', 'Jones')
+            ];
+
+        List<Id> contactIds = new List<Id>();
+        for (Contact con : contacts) {
+            contactIds.add(con.Id);
+        }
+        
+        return contactIds;
+    }       //PASS
+
 
     /**
      * Question 4
-     * Update/Uncomment the template code to create a method
      * Returns List of closed won opportunities for a specific Account.
      * Example: yourMethodName('001XXXXXXXXXXXX') should return the list of closed won opportunities for the Account with the Id
      * @param accountId The ID of the Account.
      * @return A list of Opportunities with important opportunities fields
      * Update the corresponding test class and testGetClosedWonOpportunityCountByAccount method to cover this method
-     */
-    //[public | private ] [static] data_type method_name (input parameters) {
-    //  The body of the method
-    //}
+     */    
+
+    public static List<Opportunity> getClosedWonOpportunitiesByAccount(Id accountId) {
+        List<Opportunity> closedWonOpps = [
+            SELECT Id, Name, StageName, AccountId, CloseDate, Amount
+            FROM Opportunity
+            WHERE AccountId = :accountId 
+            AND StageName = 'Closed Won'
+            ];
+        return closedWonOpps;
```

Comment:
Is it possible to return this result directly without the intermediate variable?